src/main/resources/assets/gravestone/lang/zh_cn.json

```diff
@@ -1,18 +1,34 @@
 {
-  "block.gravestone.gravestone": "墓碑",
-  "item.gravestone.obituary": "死亡笔记",
-  "entity.gravestone.player_ghost": "幽灵玩家",
-  "message.gravestone.died": "%s 死亡时间 %s",
-  "message.gravestone.deathlocation": "死亡地点",
-  "message.gravestone.no_deathlocation": "无死亡地点记录",
-  "message.gravestone.create_grave_failed": "无法放置坟墓",
-  "gui.obituary.name": "名字",
-  "gui.obituary.title": "死亡笔记",
-  "gui.obituary.location": "位置",
-  "gui.obituary.dimension": "维度",
-  "gui.obituary.title.items": "生前物品",
-  "gui.obituary.page": "页 %s",
-  "gui.obituary.time": "时间",
-  "button.gravestone.next": "下一页",
-  "button.gravestone.prev": "上一页"
+    "block.gravestone.gravestone": "墓碑",
+    "item.gravestone.obituary": "讣告",
+    "entity.gravestone.player_ghost": "玩家幽灵",
+    "message.gravestone.grave_of": "%s的墓碑",
+    "message.gravestone.date_of_death": "死于%s",
+    "message.gravestone.item_count": "包含%s个格子的物品",
+    "message.gravestone.died": "%s死于%s",
+    "message.gravestone.deathlocation": "死亡位置",
+    "message.gravestone.no_deathlocation": "没有死亡位置记录",
+    "message.gravestone.create_grave_failed": "无法生成墓碑",
+    "message.gravestone.copied": "已把%s复制到到剪贴板",
+    "message.gravestone.restore": "取回物品",
+    "message.gravestone.restore.replace": "替换",
+    "message.gravestone.restore.replace.description": "移除你背包里所有的物品并取回你上一次死亡时的物品",
+    "message.gravestone.restore.add": "添加",
+    "message.gravestone.restore.add.description": "把你上一次死亡时的物品添加入你的背包",
+    "message.gravestone.death_id": "死亡事件ID",
+    "message.gravestone.death_id_not_found": "未找到ID为%s的死亡事件",
+    "message.gravestone.restore.success": "成功取回了%s的物品",
+    "message.gravestone.death_not_found": "未找到死亡事件",
+    "gui.obituary.name": "名字",
+    "gui.obituary.title": "讣告",
+    "gui.obituary.location": "死亡位置",
+    "gui.obituary.dimension": "死亡维度",
+    "gui.obituary.title.items": "陪葬品",
+    "gui.obituary.page": "第%s页（共%s页）",
+    "gui.obituary.time": "死亡时间",
+    "gui.obituary.id": "死亡事件ID",
+    "gui.obituary.copy_id": "复制死亡事件ID",
+    "gui.gravestone.date_format": "年/月/日 时:分:秒",
```

Comment:
The date format has to be formatted like [this](https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html).